eng/common/TestResources/New-TestResources.ps1

```diff
@@ -349,7 +353,7 @@ try {
     $root = [System.IO.Path]::Combine($repositoryRoot, "sdk", $ServiceDirectory) | Resolve-Path
     $templateFiles = @()
 
-    'test-resources.json', 'test-resources.bicep' | ForEach-Object {
+    "$ResourceType-resources.json", "$ResourceType-resources.bicep" | ForEach-Object {
```

Comment:
Given my previous comment, should we fall back to "test-resources.*" if "perf-resources.*" doesn't exist so we don't have to add "perf-resources.*" to every service directory just to make this work?

Reply:
I think it will be better overall to require an explicit `perf-resources.*` if you want to have a perf pipeline.  But I don't have a strong objection to adding the fallback if others disagree.

Reply:
I don't remember off hand: does the script fail gracefully if there are no *-resources.* files then? If so, then I think your suggestion is fine.

Reply:
Are you asking what happens if you run `New-TestResources.ps1` but there are no `test-resources.json` or `test-resources.bicep` files?  I would think this should be an error, since you are trying to deploy test resources but there's no input file, right?  Unless I am misunderstanding your question.

Reply:
Also I would say if people want to use the same template for both, we should encourage them to use bicep modules as a way of doing that.

Reply:
Test pipelines run that script always. Reviewing the script, apparently I did take that into account already:

```powershell
    if (!$templateFiles) {
        Write-Warning -Message "No template files found under '$root'"
        exit
    }
```

Reply:
@mikeharder I don't think this should affect any of your changes, but it's probably worth glancing through some of the `test-resources-post.ps1` files in the other repos just in case. And will the post scripts also be discovered and run in the perf pipelines? Some I expect would break if they are tightly coupled with the test-resources.json template. E.g. net storage live tests.

Reply:
He runs "$ResourceType-resources-post.ps1", so that shouldn't matter, right?

Reply:
@heaths: Correct, I explained this to Ben offline and he agrees.  He missed this part of my change when reviewing the PR.